nservicebus/pipeline/index_intro_core_[6,8).partial.md

```diff
@@ -0,0 +1 @@
+NServiceBus has the concept of a _pipelines_ which refers to the series of actions taken when an incoming message is processed and an outgoing message is sent.
```

Comment:
```suggestion
NServiceBus has the concept of a _pipeline_ which refers to the series of actions taken when an incoming message is processed which triggers the incoming pipeline, and an outgoing message is sent which triggers the outgoing pipeline.
```
Making a suggestion here although I'm not entirely sure if it's needed. For us it's obvious, but for docs readers, it might not be. Thoughts?